praas-app-ionic/src/App.tsx

```diff
@@ -8,6 +8,7 @@ import AuthenticatedRoute from './components/AuthenticatedRoute';
 import Signup from './pages/Signup';
 import LoginPage from './pages/Login';
 import Home from './pages/Home';
+import Conduit from './pages/Conduit';
```

Comment:
what do you suggest?

Reply:
simple is as we have keep all pages in `pages` directory. or import like `HomePage`

Reply:
This is a good thought. But in code again where ever we use, this would have to be reflected. I would say to move it to `pages` directory and also to use the convention of  `HomePage` throughout